CHANGES.rst

```diff
@@ -94,6 +94,8 @@ Other Changes and Additions
 Bug Fixes
 ---------
 
+- Fix Plot Options stretch histogram's curve for non-gray colormaps. [#2537]
+
```

Comment:
since this bug was never released, can probably just append this PR number to the original feature entry

Reply:
ok, I was wrong, the curve _is_ in 3.7, but we'll need to manually backport this if we leave it here for sure...